src/webchat/helper/message.ts

```diff
@@ -1,19 +1,28 @@
-import { IMessage } from "../../common/interfaces/message"
+import { IMessage } from "../../common/interfaces/message";
 
 const getTextFromMessage = (message: IMessage) => {
+	// Check if message is plain text
+	if (message.text) {
+		return message.text;
+		// Check if message is quick reply message
+	} else if (message?.data?._cognigy?._webchat?.message?.text) {
```

Comment:
Here and down you want to use `getMessengerPayload` function and not access the `_webchat` directly as a message can come from a different channel (_facebook or _defaultPreview)

Reply:
this is a existing function I didn't change and I would not change in PR.